### PR TITLE
defining what a prize is

### PR DESCRIPTION
**Pull request by MandyMeindersma**

### CODE REVIEW

File: policies/rules.md [hunk 1]
```diff
@@ -89,11 +89,11 @@ Teams have until **Thursday, April 30th at 23:59 UTC** to submit their projects
 
 ## Problem Area Prizes (8 in total)
 
-Decided by a panel of expert judges, there are **8 prizes** up for grabs: 1 to a winner and 1 to a runner-up team in **each** of the **four [problem areas](rules#the-problems)**. Judging begins shortly after submission has closed. Although we aim to have judging wrap up within one week, we reserve the right to give our judges more time to evaluate the submissions.
+Decided by a panel of expert judges, there are **8 prizes** up for grabs: 1 to a winner and 1 to a runner-up team in **each** of the **four [problem areas](rules#the-problems)**. The runner-up prizes will consist of recognition, possibility of local news interview, marketing and some ongoing supports from organizers which will be determined on a team by team basis. The winning prizes will include all mentioned as well as a monetary prize. Judging begins shortly after submission has closed. Although we aim to have judging wrap up within one week, we reserve the right to give our judges more time to evaluate the submissions.
```
Comment (MarkBennett) [inline policies/rules.md]: Can you mention that we aim to have judging complete by May 16th instead of one week?

**Review by MarkBennett — CHANGES_REQUESTED** [review]
Looks good @MandyMeindersma! Just one small tweak.

File: policies/rules.md (hunk 1 above)
Comment (ajyong) [reply policies/rules.md]: I opted to state "mid to late-May" to give ourselves a buffer.

**Review by ajyong — APPROVED** [review]
Feels weird to approve my own commits, but honestly it was a team effort

File: policies/rules.md [hunk 2]
```diff
@@ -87,9 +87,9 @@ _This is a great opportunity to promote your project and CODEVID-19 to friends,
...
 
-## Problem Area Prizes (8 in total)
+## Problem Area Prizes (4 in total)
 
-Decided by a panel of expert judges, there are **8 prizes** up for grabs: 1 to a winner and 1 to a runner-up team in **each** of the **four [problem areas](rules#the-problems)**. The runner-up prizes will consist of recognition, possibility of local news interview, marketing and some ongoing supports from organizers which will be determined on a team by team basis. The winning prizes will include all mentioned as well as a monetary prize. Judging begins shortly after submission has closed. Although we aim to have judging wrap up within one week, we reserve the right to give our judges more time to evaluate the submissions.
+Decided by a panel of expert judges, there are **4 prizes** up for grabs: 1 to a winning team in **each** of the **four [problem areas](rules#the-problems)**. While runner-ups will not receive prizing, we will recognize one in each of the problem areas.  Although we aim to have judging wrap up around mid to late-May, we reserve the right to give our judges, sponsors and organizers enough time to evaluate submissions and work through logistics.
```
Comment (MarkBennett) [inline policies/rules.md]: Looks perfect. Now to get the lead judges sorted!

**Review by MarkBennett — APPROVED** [review]

**Comment by MandyMeindersma** [comment]
lol I can't approve this cause it "my" pr

**Comment by MandyMeindersma** [comment]
but lgtm

**Comment by MarkBennett** [comment]
So that this is live, I've taken the liberty of merging as it's been approved @MandyMeindersma. Sorry, hope that's ok!

**Pull request merged**